{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py

```diff
@@ -50,6 +50,34 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# Content Security Policy
+if env('CSP_ENABLED') == 'y':
+    MIDDLEWARE.append('csp.middleware.CSPMiddleware')
+
+    CSP_ENABLED = True
+
+    CSP_REPORT_ONLY = env('CSP_REPORT_ONLY', default='y') == 'y'
+    CSP_REPORT_URL = env('CSP_REPORT_URL', default=None) or None
+
+    env_csp_src = lambda name: tuple(env(name, default='').split(','))
```

Comment:
use `env.tuple` or `env.list` instead of own lambda functions. Also, it's not good idea to assign lambda functions to another variable.